### PR TITLE
feat: catalog

### PR DESCRIPTION
**Pull request by Aganivi**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [x] This change requires follow-up update in localized docs.

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const { execSync } = require('node:child_process')
+const settingData = require('./setting.json')
+
+// TODO update address before pop up online (update websiteMap in setting.json)
```
Comment (nicecui) [inline script/build.js:5]: what does it mean? do we need to update the address manually?

Comment (Aganivi) [reply script/build.js:5]: It has been updated, just to remind myself

File: docs/zh/v0.3/user-guide/concepts/architecture.md
Comment (xtang) [inline docs/zh/v0.3/user-guide/concepts/architecture.md:1]: is this an accidental deletion?

**Review by nicecui — APPROVED** [review]

File: docs/zh/v0.3/user-guide/concepts/architecture.md
Comment (Aganivi) [reply docs/zh/v0.3/user-guide/concepts/architecture.md:1]: It was moved to 'docs/v0.3/zh/user-guide/concepts/architecture.md'

**Review by xtang — APPROVED** [review]

**Pull request merged**